src/pages/studyView/StudyViewUtils.spec.ts

```diff
@@ -151,7 +151,7 @@ describe('StudyViewUtils', () => {
                     }]
                 }],
                 mutatedGenes: [{ "hugoGeneSymbols": ["GENE1"] }],
-                fusionGenes: [{ "entrezGeneIds": [1] }],
+                fusionGenes: [{ "hugoGeneSymbols": ["GENE1"] }],
```

Comment:
Should we use a different gene for fusion?

Reply:
I think it's fine.